lib/Bailador/Configuration.pm

```diff
@@ -30,6 +35,35 @@ class Bailador::Configuration {
                 }
             }
         }
+    }
+
+    method load-from-file() {
+        unless $.config_file.IO.e {
+            warn "The configuration file wasn't found.";
+            warn "Bailador will use his default configuration.";
+        }
 
+        if $.config_file.IO.extension ~~ 'yaml' | 'yml' {
+            try {
+                my $yaml = slurp $.config_file;
+                my %config = load-yaml($yaml);
+                for %config.kv -> $k, $v {
```

Comment:
Could you think of a more generic way to set the member variables of the Bailador::Configuration? it would be really awesome if you dont need to add a new `when 'whateve'` for every new parameter that might be added in the future.

Thanks :-)